File: Domains/ApplicationDomain/BOA/Models/Farmers/FarmerModel.cs
using ApplicationDomain.BOA.Entities;
using ApplicationDomain.Identity.Entities;
using AutoMapper;
using System;

namespace ApplicationDomain.BOA.Models.Farmers
{
    public class FarmerModel
    {
        public int Id { set; get; }
        public string Name { set; get; }
        public string Code { set; get; }
        public string FullName { get; set; }
        public string PhoneNumber { get; set; }
        public string Address { get; set; }
        public DateTime Birthday { get; set; }
        public bool Gender { get; set; }
        public int? DistrictId { set; get; }
        public int? ProvinceId { get; set; }
        public bool IsBlock { get; set; }
        public DateTime ContractCreatetionDate { get; set; }
        public int? UserId { get; set; }
        public int? MilkCollectionStationId { get; set; }
        public int? WareHouseId { get; set; }
        public string IdentificationNo { get; set; }
        public DateTime IssuedOn { get; set; }
        public string IssuedBy { get; set; }
        public string AccountNumber { get; set; }
        public string Bank { get; set; }
        public string UserUserName { get; set; }
        public string BankBranch { get; set; }
        public string Landline { get; set; }
    }

    public class FarmerModelMapper : Profile
    {
        public FarmerModelMapper()
        {
            CreateMap<Farmer, FarmerModel>();
        }
    }
}

File: Applications/MobileFarmerApplication/lib/home/model/farmer_model.dart
class FarmerInfoModel {
  int id;
  String name;
  String code;
  String fullName;
  String email;
  String phoneNumber;
  String avatarURL;
  String address;
  bool status;
  int userId;
  String userUserName;
  bool gender;
  DateTime birthday;


  FarmerInfoModel({
    this.id,
    this.userId,
    this.name,
    this.code,
    this.birthday,
    this.address,
    this.gender,
    this.fullName,
    this.email,
    this.userUserName,
    this.phoneNumber,
    this.avatarURL,
    this.status,
  });

  FarmerInfoModel.fromJson(Map<String, dynamic> json) {
    id = json['Id'];
    name = json['Name'];
    code = json['Code'];
    address = json['Address'];
    birthday = DateTime.parse(json['Birthday']);
    fullName = json['FullName'];
    userUserName = json['UserUserName'];
    gender = json['Gender'];
    email = json['Email'];
    userId = json['userId'];
    phoneNumber = json['PhoneNumber'];
    avatarURL = json['AvatarURL'];
    status = json['Status'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = new Map<String, dynamic>();
    data['Id'] = this.id;
    data['Name'] = this.name;
    data['Email'] = this.email;
    data['UserUserName'] = this.userUserName;
    data['Birthday'] = this.birthday;
    data['UserId'] = this.userId;
    data['Address'] = this.address;
    data['Code'] = this.code;
    data['Fullname'] = this.fullName;
    data['PhoneNumber'] = this.phoneNumber;
    data['AvatarURL'] = this.avatarURL;
    data['Status'] = this.status;
    return data;
  }
}
